leanix_admin/model.py

```diff
@@ -0,0 +1,42 @@
+import leanix_admin.action as action
+import leanix_admin.file as file
```

Comment:
Same remark about imports: I think it's more idiomatic to use `from leanix_admin import action`; at least in Python 2 it was 😉